components/brave_shields/resources/panel/components/main-panel/style.ts

```diff
@@ -65,16 +65,16 @@ export const SiteTitle = styled.h1`
 
 export const CountBox = styled.section`
   display: grid;
-  grid-template-columns: 24px 2fr 0.5fr;
+  grid-template-columns: 24px 2fr 80px;
   align-items: center;
   grid-gap: 10px;
+  overflow: hidden;
```

Comment:
what does the `overflow: hidden` do? Since we're restricting the count string length, is a specific size needed, or can it be auto? Just wondering about whether other fonts will fit perfectly or need more space

Reply:
i'd say this is a left over. before restricting the number I tried various ways to hide it and overflow was one of them.